### PR TITLE
add quotes to tutorial cd

### PR DESCRIPTION
**Pull request by shaycrk**
I have a space in my directory name, so this cd was failing without quotes around `$INFRASTRUCTURE_HOME`

### CODE REVIEW

**Comment by shaycrk** [comment]
(also some SQL debugging here...)

File: src/sql/create_semantic_tables.sql [hunk 1]
```diff
@@ -1,41 +1,52 @@
 create schema if not exists semantic;
```
Comment (nanounanue) [inline src/sql/create_semantic_tables.sql]: Could you remove the sql files from the *PR*? The sql files were already generated to reflect the org file.

File: triage/example_database.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 host: food_db
-pass: some_password
+pass: your_password
```
Comment (nanounanue) [inline triage/example_database.yaml]: Thank you!

Hopefully this file will dissapear when `triage` CLI supports reading from the environment

File: tutorial.sh [hunk 3]
```diff
@@ -6,7 +6,7 @@ PROJECT="$(cat .project-name)"
 PROJECT_HOME="$( cd "$( dirname "$0" )" && pwd )"
 INFRASTRUCTURE_HOME="${PROJECT_HOME}/infrastructure"
 
-cd $INFRASTRUCTURE_HOME
+cd "$INFRASTRUCTURE_HOME"
```
Comment (nanounanue) [inline tutorial.sh]: :+1:

**Review by nanounanue — CHANGES_REQUESTED** [review]

**Review by nanounanue — APPROVED** [review]

**Pull request merged**